src/components/deploymentConfig/DeploymentConfig.utils.ts

```diff
@@ -185,3 +186,47 @@ export const getPosition = (isValues: boolean, isEnv: boolean, type: number) =>
         if (type === 2) return 2
     }
 }
+
+function removeEmptyValues(obj) {
```

Comment:
Please extract these methods in common-lib so that it can be reused

Reply:
done